### PR TITLE
Password boilerplate html

### PR DESCRIPTION
**Pull request by AndyZng**
Merging to add error message functionality

### CODE REVIEW

File: src/password/landing.js [hunk 1]
```diff
@@ -12,9 +12,14 @@ document.getElementById('input-password-form').addEventListener('submit', async
 
     //TODO: show incorrect password error on the page
     else{
-        console.log("incorrect password")
+        const errorMessage = document.getElementById('error-message');
+        errorMessage.style.display = 'block';
     }
 })
+document.getElementById('try-again-button').addEventListener('click', function(){
+    const errorMessage = document.getElementById('error-message');
```
Comment (ravishende) [inline src/password/landing.js:20]: When we select an element from js, we've been using `document.querySelector('js-{class-name}')`, where {class-name} is the rest of the class name. This is so we can still have js functionality even if css class names/ids change.

So, for this, you would want to add a class to the element called 'js-error-message', then say `document.querySelector('.js-error-message')`. Same goes for line 19, with a new class for 'js-try-again-button'.

Also, for line 19, I would recommend saying `const tryAgainBtn = document.getElementById('js-try-again-button');`, then on a new line, say `tryAgainBtn.addEventListener(...)`

File: src/password/landing.css [hunk 2]
```diff
@@ -46,3 +46,31 @@ button[type="submit"]:hover {
     color: #21528B;
 }
 
+.error{
+    font-size: 16px; /* Smaller font size */
+    display: none; /* Hidden by default */
+    color: red;
+    background-color: #f8d7da;
```
Comment (ravishende) [inline src/password/landing.css:53]: change colors and numbers to be in css variables

File: src/settings/settings.html [hunk 3]
```diff
@@ -26,6 +29,10 @@ <h1>Settings</h1>
                 <input type="password" id="confirm-password" name="confirm-password" required>
                 <button type="submit">Update</button>
             </form>
+            <div id="error-message" class="error">
+                <span id="error-text">Passwords do not match. Please try again.</span>
+                <button id="try-again-button">Try Again</button>
+            </div>
             <script src="settings.js"></script>
             <script>
```
Comment (ravishende) [inline src/settings/settings.html:37]: Move this script into settings.js

**Review by ravishende — APPROVED** [review]
Looks good for the most part, but there are some things that need changing. Please read the comments I left and implement them, then create another PR. You can merge this tho.

File: src/password/landing.js (hunk 1 above)
Comment (AndyZng) [reply src/password/landing.js]: Hi I fixed all the issues and pushed again to password boiler plate. It's not letting me PR again for some reason though

**Pull request merged**